tests/test_custom_row_selectable.py

```diff
@@ -0,0 +1,127 @@
+import dash_ag_grid as dag
```

Comment:
How about making a simpler test? Maybe something like:

```

import dash_ag_grid as dag
from dash import Dash, html

def test_cr002_custom_row_selectable(dash_duo):

    app = Dash(__name__)

    columnDefs = [
        {    
            "field": "ticker",
            "headerCheckboxSelection": True,
            "checkboxSelection": True,
            "showDisabledCheckboxes": True,
        }
    ]
    grid = dag.AgGrid(
        id="grid",
        columnDefs=columnDefs,
        rowData=[{"ticker": "AAPL"}, {"ticker": "MSFT"}],
        dashGridOptions={'isRowSelectable': {'function': 'params.data.ticker == "AAPL" ? true: false'}},
        rowSelection="multiple"
    )

    app = Dash(__name__)

    app.layout = html.Div(grid)

    dash_duo.start_server(app)

    grid = utils.Grid(dash_duo, "grid")

    grid.wait_for_cell_text(0, 1, "Apple")

    assert dash_duo.find_element('#grid [row-index="0"] [aria-colindex="1"] '
                                 '.ag-selection-checkbox input').get_attribute('disabled') == None
    assert dash_duo.find_element('#grid [row-index="1"] [aria-colindex="1"] '
                                 '.ag-selection-checkbox input').get_attribute('disabled')


```

Reply:
That’s fine for me.

Reply:
See https://github.com/plotly/dash-ag-grid/pull/67/commits/8bdd11d8ff2d513ad32c88467504aeb42d3f87a9